api/user/schema.py

```diff
@@ -33,6 +33,8 @@ class Arguments:
 
     def mutate(self, info, **kwargs):
         user = UserModel(**kwargs)
+        if not verify_email(user.email):
+            raise GraphQLError("Can\'t add a Non-Andelan email")
```

Comment:
Use 'This email is not allowed' instead of 'Can't add a Non-Andelan email' just for security purposes. We don't want to suggest to intruders what email is allowed to our database.

Reply:
Kindly check on your commit messages. Please use the guide that I shared on the slack channel last weeks